src/Symfony/Bundle/TwigBundle/Tests/Controller/ExceptionControllerTest.php

```diff
@@ -19,7 +19,7 @@ class ExceptionControllerTest extends TestCase
 {
     public function testOnlyClearOwnOutputBuffers()
     {
-        $flatten = $this->getMock('Symfony\Component\HttpKernel\Exception\FlattenException');
+        $flatten = $this->getMock('Symfony\Component\Debug\Exception\FlattenException');
```

Comment:
This should be reverted. The ExceptionController is typehinted against the deprecated interface and this cannot be changed for BC reasons (changing the signature is a BC break for people extending it, and this controller is known to be extended. Many people complained because of the BC break when we tried changing this signature in the past)